tests/symmetry/test_groups.py

```diff
@@ -198,14 +205,9 @@ def test_subgroup_supergroup(self):
         assert SpaceGroup("Pma2").is_subgroup(SpaceGroup("Pccm"))
         assert not SpaceGroup.from_int_number(229).is_subgroup(SpaceGroup.from_int_number(230))
 
-    def test_hexagonal(self):
-        for num in (146, 148, 155, 160, 161, 166, 167):
-            sg = SpaceGroup.from_int_number(num, hexagonal=False)
-            assert not sg.symbol.endswith("H")
```

Comment:
why remove this test?

Reply:
I removed it as the representation ("H" as last character) is not part of the symbol attribute in any case now.
We could maybe still keep this test and replace the assertion with a test for the hexagonal attribute `assert sg.hexagonal == False` to check that this is given correctly to the SpaceGroup init.

Reply:
replacing the assertion with a test for the hexagonal attribute sounds good